README.md

```diff
@@ -1,6 +1,6 @@
-# hw01
+# Homework assigment 1
 #Bio of Mary Denise Robertson
```

Comment:
Prof Soltoff told me that putting a space between the hashtag and first letter makes it so the hashtag doesn't transfer over to the final view (I did the exact same thing).

Reply:
Thank you, that makes sense! Cool that another anthropologist is taking the class :-) Actually, the milkshakes are a dollar, not free, and they are at Pret in Reynolds - or at least they were.